components/Panel/Panel.css

```diff
@@ -1,6 +1,5 @@
 .root {
+  composes: bs br from '../../globals/modifiers.css';
```

Comment:
It won't show directly — it basically maps `bs` and `br` from modifiers onto the Panel root.

so:
```css
.root {
  composes: br bs from 'modifiers.css';
  background: red;
}
```
outputs
```css
.Panel-root {
  background: red;
}
.modifiers-bs, .Panel-root {
  border-radius: 8px;
}
.modifiers-br, .Panel-root {
  box-shadow: whatever
}
```